app/controllers/challenges_controller.rb

```diff
@@ -19,6 +20,26 @@ def create
     end
   end
 
+  def for_evaluation
+    @evaluable_challenges = Challenge.evaluable(current_user)
```

Comment:
I agree though I can't think of a better one.. The internet says evaluatable is [not even close to evaluable](https://english.stackexchange.com/questions/304948/evaluable-vs-evaluatable) in usage. So probably not actually a word